test/ibmq/runtime/test_runtime.py

```diff
@@ -501,7 +501,7 @@ def test_cancel_job(self):
         """Test canceling a job."""
         job = self._run_program(job_classes=CancelableRuntimeJob)
         time.sleep(1)
-        job.cancel()
+        cancel_job(job)
```

Comment:
nit - `cancel_job` takes an `IBMQJob`, but the job here is a `RuntimeJob`. It's a nit because the syntax is the same for both (for now anyway), you just won't get the benefit of using the method (e.g. it'd never throw a `JobError`).